src/synthesizer/utils.py

```diff
@@ -102,6 +102,134 @@ def rebin_1d(arr, resample_factor, func=np.sum):
     return func(brr, axis=1)
 
 
+def process_dl07_to_hdf5(grid_name='MW3.1', grid_loc='./',
```

Comment:
This feels oddly placed. It should probably be in the grids repo.

Reply:
Could add a `dust` module for it. Since you're effectively making a grid here.

Reply:
No problem. I can remove it from here and add it in the grids repo.